controllers/ocirepository_controller.go

```diff
@@ -898,7 +935,8 @@ func (r *OCIRepositoryReconciler) eventLogf(ctx context.Context,
 
 // notify emits notification related to the reconciliation.
 func (r *OCIRepositoryReconciler) notify(ctx context.Context,
-	oldObj, newObj *sourcev1.OCIRepository, res sreconcile.Result, resErr error) {
+	oldObj, newObj *sourcev1.OCIRepository, res sreconcile.Result, resErr error,
+) {
```

Comment:
Above changes seem to be unrelated unintentional changes.

Reply:
reverted, thanks.